packages/lit-element/src/lib/updating-element.ts

```diff
@@ -688,11 +688,19 @@ export abstract class UpdatingElement extends HTMLElement {
   }
 
   /**
-   * This protected version of `requestUpdate` does not access or return the
-   * `updateComplete` promise. This promise can be overridden and is therefore
-   * not free to access.
+   * Requests an update which is processed asynchronously. This should
+   * be called when an element should update based on some state not triggered
+   * by setting a property. In this case, pass no arguments. It should also be
```

Comment:
```suggestion
   * by setting a reactive property. In this case, pass no arguments. It should also be
```

I've started to standardize on calling our special properties "reactive properties". We had been calling them "declared properties", but that doesn't seem to distinguish them correctly to me. @arthurevans has been considering standardizing across docs I think.